src/bp/ui-admin/src/App/CommandPalette.tsx

```diff
@@ -0,0 +1,61 @@
+import { Commander, lang, QuickShortcut } from 'botpress/shared'
+import React, { useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+
+import { fetchBots } from '../reducers/bots'
+
+const CommandPalette = props => {
```

Comment:
We don't have the same typings in admin than we have in the studio, so i'll leave it as-is in this PR. We'd need to type reducers and other stuff, it could be in its own PR (lot of stuff to change)